userspace/libsinsp/test/events_proc.ut.cpp

```diff
@@ -719,23 +719,19 @@ TEST_F(sinsp_with_test_input, spawn_process) {
 	// check that the name is updated
 	ASSERT_EQ(get_field_as_string(evt, "proc.name"), "test-exe");
 	ASSERT_EQ(get_field_as_string(evt, "proc.aname[0]"), "test-exe");
-	ASSERT_EQ(get_field_as_string(evt, "proc.aname"), "test-exe");
```

Comment:
Question: why did you remove all these lines in this test file?
Is it not supported anymore to call these filterchecks without an arg?

Reply:
I am working on a new test, gonna push soon :)

Reply:
> Is it not supported anymore to call these filterchecks without an arg?

Fun fact, _printing_ any of those fields with no argument was never supported. The tests were not correct.

Reply:
(see this issue: https://github.com/falcosecurity/libs/issues/2229 )

Reply:
Thanks! This does not surprise me at all 😂